docs/search/get-started-with-search/search-page/log-message-inspector.md

```diff
@@ -0,0 +1,27 @@
+---
+id: log-message-inspector
+title: Log Message Inspector
+sidebar: Log Message Inspector
+description: You can highlight a time range in the histogram for your search results to filter the search results based on that time range.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By default, when performing Log Search queries, the results are displayed in the **Time** and **Messages** column. You can further drill down the results using [Show and Hide Fields in the Field Browser](/docs/search/get-started-with-search/search-page/field-browser/show-hide-fields-in-field-browser).
```

Comment:
`when performing Log Search queries  -> When performing Non-aggregate Log Search queries.`
We should include this information somewhere, stating that the Log Inspector view is only available on the Message tab.

Reply:
I will add it.